HeterogeneousCore/CUDAUtilities/test/prefixScan_t.cu

```diff
@@ -136,10 +136,13 @@ int main() {
     cudaCheck(cudaMalloc(&d_pc, sizeof(int32_t)));
     cudaCheck(cudaMemset(d_pc, 0, 4));
 
-    nthreads = 1024;
+    nthreads = 512+256;
```

Comment:
problem  understood ("-G" in the Buildfile) can go back to be 1024

Reply:
could you undo the change, then ?

Reply:
ok, will try "above" pre7

Reply:
I've updated the target branch in GitHub to avoid pulling it the whole history (I don't know if there is a simpler way to do it)